### PR TITLE
location gltravel: add example config for travelrouter....

### PR DESCRIPTION
**Pull request by Petrella**
…with glinet mt1300 as router

### CODE REVIEW

**Comment by FFHener** [comment]
Please do a `rebase` on your PR to squash the Mergecommit.

File: locations/gltravel.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+  # UPLK
+  - vid: 50
+    role: uplink
+    untagged: true
+
+  - role: tunnel
+    ifname: ts_wg0
+    mtu: 1280
+    prefix: 10.31.176.35/32
```
Comment (pktpls) [inline locations/gltravel.yml:122]: You're already using .35 for mesh_lan

**Comment by pktpls** [comment]
LGTM except for the tunnel/mesh_lan IP duplication :+1:

**Comment by pktpls** [comment]
Bitte wenn es soweit ist den PR wieder aufmachen

**Pull request closed without merging**